Healthy/Classes/Modules/Login/LoginViewModelType.swift

```diff
@@ -1,9 +1,20 @@
+import Foundation
+
 typealias LoginViewModelType = LoginViewModelInput & LoginViewModelOutput
 
-protocol LoginViewModelInput {
+// MARK: Actions
 
+protocol LoginViewModelInput {
+    func updateEmail(_ text: String)
+    func updatePassword(_ text: String)
 }
 
-protocol LoginViewModelOutput {
+// MARK: Updates
 
+protocol LoginViewModelOutput {
+    func bindLoadingIndicator(state: @escaping (Bool) -> Void)
+    func bindErrorMessage(message: @escaping (String) -> Void)
+    func bindButtonEnabled(onEnabled: @escaping (Bool) -> Void)
+    func bindLoginStatus(status: @escaping (Bool) -> Void)
```

Comment:
We won't depend on closures for the output. But this is a good starting point.

Reply:
Can you apply combine for this? Or do ew need to pair on it?

Reply:
okay i will read about combine and i will apply it.

Reply:
Can we have a meeting to discuss this?

Reply:
okay